index.html

```diff
@@ -1,21 +1,93 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
 <head>
-  <meta charset="UTF-8">
+  <meta charset='UTF-8'>
   <title>Does the CSSU have a microwave?</title>
   <style>
-    h1 {
+    #yes {
       font-size: 6em;
       font-weight: 700;
       margin: 0;
       position: absolute;
       top: 50%;
       left: 50%;
       transform: translate(-50%, -50%);
+      text-align: center;
+    }
+
+    #since {
+      font-size: 0.15em;
+      margin: 0 auto;
+      text-align: center;
     }
   </style>
 </head>
 <body>
-  <h1>YES.</h1>
+  <div id='yes'>
+    YES.
+    <p id='since'></p>
+  </div>
+  <script>
+    //TODO: update this on commit automatically
+    var LAST_UPDATE = 1483628415000;
```

Comment:
Might be worth leaving a comment noting what this value is (i.e. the old string value).